pkg/virtctl/vm/vm.go

```diff
@@ -332,6 +334,13 @@ func (o *Command) Run(args []string) error {
 
 	switch o.command {
 	case COMMAND_START:
+		if startPaused {
+			err = virtClient.VirtualMachine(namespace).StartPaused(vmiName)
+			if err != nil {
+				return fmt.Errorf("Error starting VirtualMachine %v", err)
+			}
+			break
+		}
```

Comment:
typically the pattern we see in the k8s ecosystem is we can have an api call, like `Delete` for example, which then has a set of options that can be passed in `metav1.DeleteOptions{}`.

We follow this pattern in our API as well. Take a look at the `RemoveVolume` api with the `v1.RemoveVolumeOptions` 

I'd suggest rather than a new `StartPaused`  api, use the existing `Start` api and add the StartOptions similar to how the other apis work

Reply:
We should be cautious with such change since it might be a breaking change and (I think) this package is aimed at external users (via kubevirt/client-go).
We can make it a variadic function `Start(name string, ...v1.StartOptions)` but that's also diverts from the pattern.

Reply:
@davidvossel ^^

Reply:
here we care about api compatibility, and not necessarily ensuring that the functions within our exported packages remain consistent.

So, someone should be able to use an old "start" client command and have it still work with a new kubevirt. It's okay if updating the package in their repository requires code changes though

Reply:
Changed